### ISSUE TITLE
Add option to drop callouts numbers

### ISSUE THREAD

**Issue by mojavelinux** [asciidoctor/asciidoctor#3333]
Sometimes, you want to display a verbatim block (such as a source listing) without the callout numbers. For example, in one case you may use the callout numbers to explain how the source works, and in another case you just want to show the source as it will be written. This could be accomplished by adding an option to to the block to drop the callouts.

Possible names for this option are as follows:

* nocallouts
* purify

Feel free to suggest others.

Here's an example:

```
[%nocallouts,ruby]
----
include::hello-world.rb[]
----
```

(Naturally, the source would be coming from an include file. Otherwise, this option wouldn't make much sense).

Another approach to take would be to add this option to the include directive. That would fit well since there's likely a 1-to-1 mapping between using this option and using the include directive.

**Comment by elextr** [comment]
`nocallouts` seems fine for a delimited block, its explicit whereas purify could have many other meanings.  

If its part of the include options what is it going to apply to?  Since the include (by definition) is a preprocessor and does not know about structure.  Efffectively its introducing the ability to apply particular options to a region of the document irrespective of the underlying structure.  That capability would likely be useful for other options, but its potentially tricky to apply when the document has been parsed from the linear text to a tree structure.

**Comment by mojavelinux** [comment]
cc: @graphitefriction

**Comment by mojavelinux** [comment]
> Since the include (by definition) is a preprocessor and does not know about structure.

That's true, but callouts are also blind to context, so the combination works. The removal would do it blindly (assuming what you are doing is including some sort of source code).

**Comment by nadlertz** [comment]
> Any chance this gets implemented soon? https://stackoverflow.com/questions/69698459/disable-callout-parsing-in-asciidoctor
> 
> I would also vote for `nocallouts`.
> 
> Are there any workarounds?

I think you can do:

```
[source,subs="-callouts"]
....
x<32>
....
```

and it will be shown as
```
x<32>
```
